app/harvesters/ia_harvester.rb

```diff
@@ -0,0 +1,178 @@
+require 'json'
+
+##
+# A harvester implementation for Internet Archive
+#
+class IaHarvester
+  include Krikri::Harvester
+
+  DEFAULT_THREAD_COUNT = 10
+  DEFAULT_MAX_RECORDS = 0
+
+  DOWNLOAD_BASE_URI = 'http://archive.org/download'
+
+  ##
+  # Initialize, and set default options as appropriate.
+  #
+  # @param opts [Hash] a hash of options as defined by {.expected_opts}
+  #
+  # @example
+  #    Typical instantiation, good for most cases:
+  #
+  #      IaHarvester.new(uri: 'http://archive.org/...')
```

Comment:
Also related to the `uri` parameter, we harvest specific collections from Internet Archive, which we specify with a querystring parameter as follows:
```
?q=collection%3A%28blc%20OR%20cambridgepubliclibrary%20OR%20clemson%20OR%20bostonpubliclibrary%20OR%20regionaldigitizationmass%20OR%20getty%20OR%20medicalheritagelibrary%20OR%20MontanaStateLibrary%20OR%20yivoinstitutelibrary%20OR%20guggenheimlibrary%20OR%20lbiperiodicals%20OR%20frickartreferencelibrary%29
```
... which decodes to:
```
collection:(blc OR cambridgepubliclibrary OR clemson OR bostonpubliclibrary OR regionaldigitizationmass OR getty OR medicalheritagelibrary OR MontanaStateLibrary OR yivoinstitutelibrary OR guggenheimlibrary OR lbiperiodicals OR frickartreferencelibrary)
```

As it is now, we have to construct the URI and paste it into the `uri`, including that long string for the collections. It would be ideal to have this constructed for us if we pass in a `collections` array parameter.